src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -975,7 +975,7 @@ public String getResourcePath() {
     public boolean resourceNotAvailable() {
         getIgnoredNames();
         return getResourcePath().equals("/") || ignoredNames.ignore(getPath())
-                || ignoredNames.ignore(resourceFile.getParentFile().getName())
```

Comment:
The intention is to distinguish between ignored file check and ignored dir check. In this case the parent file should be always a directory, which should be correctly identified by the ignoredNames class (and only the directory test should be performed). Using File object instead of the String basename solves this problem.